l10n_th_vat_report/README.rst

```diff
@@ -0,0 +1,99 @@
+===================================
+Thailand Localization - VAT Reports
+===================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--thailand-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-thailand/tree/12.0/l10n_th_vat_report
+    :alt: OCA/l10n-thailand
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-thailand-12-0/l10n-thailand-12-0-l10n_th_vat_report
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/238/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+VAT Report is the report that display transactions of the selected Tax type in a specific date range (period).
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Invoicing > Reporting > VAT Report.
+#. Select Tax, Period.
+#. Choose View or Print PDF or Export XLSX or Cancel.
+
+Changelog
+=========
+
+12.0.1.0.0 (2019-08-05)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* Start of the history
+* Add Thai VAT Report
+
+12.0.1.0.1 (2019-08-08)
```

Comment:
We don't need the history or new version yet (we never have merged the first one yet).